contracts/BatchExchange.sol

```diff
@@ -31,6 +31,9 @@ contract BatchExchange is EpochTokenLocker {
     /** @dev minimum allowed value (in WEI) of any prices or executed trade amounts */
     uint128 public constant AMOUNT_MINIMUM = 10**4;
 
+    /** @dev Numerator or denominator used in orders, which do not track its usedAmount*/
+    uint128 public constant PRICE_NUMERATOR_OR_DENOMINATOR_FOR_ORDERS_NOT_TRACKING_USED_AMOUNTS = uint128(-1);
```

Comment:
We should call this
```
uint128 public constant KOEPPELMANN_CONSTANT = uint128(-1);
```
:stuck_out_tongue_closed_eyes:

Reply:
Or maybe just `MAX_U128`

Reply:
I think `MAX_U128` fails to capture that when this specific magic value is used, then it means something completely different and is not an actual number but a flag. I do agree that it could be shorter though :)

Reply:
I agree that the name is very long :(, but I could not come up with anything shorter that would still enable everyone to guess what it is.
If anyone has a proposal, I am happy to adapt it

Reply:
Can't think of anything either.

Reply:
How about UNLIMITED_AMOUNT_ORDER or UNLIMITED_ORDER_AMOUNT?

Reply:
+1 for `UNLIMITED_ORDER_AMOUNT`